host_core/lib/host_core/control_interface/lattice_server.ex

```diff
@@ -114,19 +114,13 @@ defmodule HostCore.ControlInterface.LatticeServer do
     end
   end
 
-  # Remove a link definition
-  # This will first remove the link definition from memory, then publish the removal
-  # message to the stream, then publish the removal directly to the relevant provider via the
-  # RPC channel
   defp handle_request({"linkdefs", "del"}, body, _reply_to, prefix) do
     Tracer.with_span "Handle Linkdef Del (ctl)", kind: :server do
       with {:ok, ld} <- Jason.decode(body),
            true <- has_values(ld, ["actor_id", "contract_id", "link_name"]) do
         HostCore.Linkdefs.Manager.del_link_definition(
           prefix,
-          ld["actor_id"],
-          ld["contract_id"],
-          ld["link_name"]
+          ld["id"]
```

Comment:
This requirement for an ID on the delete link definition request breaks the wasmCloud dashboard and the wash ctl command. I'd suggest either sticking to actor/contract/link or using that as a fallback if the ID is not provided.

For example, the payload from `wash` for a link del is 
```
{"actor_id":"MB2ZQB6ROOMAYBO4ZCTFYWN7YIVBWA3MTKZYAQKJMTIHE2ELLRW2E3ZW","provider_id":"","link_name":"default","contract_id":"wasmcloud:httpserver","values":{}}
```